src/js/pages/Home.js

```diff
@@ -11,16 +20,22 @@ class Home extends Component {
       title: 'Lady Bird Lake walking trail',
     }
 
+    const bannerContent = () => (
+      <p>
+        Alpha.austin.gov is a new website and a work in progress. For the full City of Austin website, visit <ExternalLink to="https://austintexas.gov">austintexas.gov</ExternalLink>. Learn more about the new website at <ExternalLink to="https://bit.ly/atx-digital-services">bit.ly/atx-digital-services</ExternalLink>.
+      </p>
+    )
+
 		return (
 			<div>
         <Hero home={true} image={homepageImage}>
           <span className="coa-Hero__home-preheader">Welcome to</span>
           <h2 className="coa-Hero__home-header">Austin, TX</h2>
           <GlobalSearch placeholder="Search for services" />
         </Hero>
+        <SecondaryContentBanner content={bannerContent()} />
 	      <section className="wrapper coa-section">
-	        <h1> Home Page </h1>️
-	        <p>just a placeholder</p>
+          <Service311 services311={services311} />
```

Comment:
The 311 module is already wrapped in a wrapper and section, so I don't think we need line 37 here and 39 to wrap it again.